src/main/cliapp/src/routes.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { SecureRoute } from '@okta/okta-react';
+import { oktaSignInConfig } from './oktaAuthConfig';
+
+import { DashboardPage } from './containers/dashboardPage';
+import { Login } from './Login'
+
+import { DataLoadsPage } from './containers/dataLoadsPage/';
+import { DiseaseAnnotationsPage } from './containers/diseaseAnnotationsPage';
+import { AffectedGenomicModelPage } from './containers/affectedGenomicModelPage';
+import { AllelesPage } from './containers/allelesPage';
+import { GenesPage } from './containers/genesPage';
+import { MoleculesPage } from './containers/moleculesPage';
+import { ControlledVocabularyPage } from './containers/controlledVocabularyPage';
+
+import { FMSComponent } from './components/FMSComponent';
+
+
+import { CHEBIOntologyComponent } from './containers/ontologies/CHEBIOntologyComponent';
+import { DiseaseOntologyComponent } from './containers/ontologies/DiseaseOntologyComponent';
+import { ECOOntologyComponent } from './containers/ontologies/ECOOntologyComponent';
+import { GOOntologyComponent } from './containers/ontologies/GOOntologyComponent';
+import { SOOntologyComponent } from './containers/ontologies/SOOntologyComponent';
+import { MAOntologyComponent } from './containers/ontologies/MAOntologyComponent';
+import { ZFAOntologyComponent } from './containers/ontologies/ZFAOntologyComponent';
+import { MPOntologyComponent } from './containers/ontologies/MPOntologyComponent';
+import { DAOOntologyComponent } from './containers/ontologies/DAOOntologyComponent';
+import { EMAPAOntologyComponent } from './containers/ontologies/EMAPAOntologyComponent';
+import { WBbtOntologyComponent } from './containers/ontologies/WBbtOntologyComponent';
+import { XCOOntologyComponent } from './containers/ontologies/XCOOntologyComponent';
+import { ZECOOntologyComponent } from './containers/ontologies/ZECOOntologyComponent';
+
+
+export default (
+  <>
+    <Route path='/login' render={() => <Login config={oktaSignInConfig} />} />
+    <SecureRoute path="/" exact component={DashboardPage} />
```

Comment:
@adamgibs does this route need to be at the bottom so that is becomes a "catch-all" route?